include/argparse/argparse.hpp

```diff
@@ -457,7 +457,7 @@ class Argument {
       return start;
     } else if (mNumArgs <= std::distance(start, end)) {
       if (auto expected = maybe_nargs()) {
-        end = std::next(start, *expected);
+        end = std::next(start, static_cast<long>(*expected));
```

Comment:
because second argument for `std::next` is `typename std::iterator_traits<InputIt>::difference_type`. where `InputIt` difference_type is long int. In that case `InputIt` is `it` variable on this line:  
https://github.com/p-ranav/argparse/blob/master/include/argparse/argparse.hpp#L1074

Reply:
I see the need, now. But, [MSVC++ has 32-bit long int.](https://docs.microsoft.com/en-us/cpp/build/common-visual-cpp-64-bit-migration-issues) The safer cast appears to be to `decltype(it)::difference_type`.